ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1036,20 +1036,38 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     ///
     /// - Parameters:
     ///   - title: Text that appears as the first line of text
+    ///   - attributedTitle: Optional attributed text for the first line of text. If this is not set, the title will be used
     ///   - subtitle: Text that appears as the second line of text
+    ///   - attributedSubtitle: Optional attributed text for the second line of text. If this is not set, the subtitle will be used
     ///   - footer: Text that appears as the third line of text
+    ///   - attributedFooter: Optional attributed text for the third line of text. If this is not set, the footer will be used
     ///   - customView: The custom view that appears near the leading edge next to the text
     ///   - customAccessoryView: The view acting as an accessory view that appears on the trailing edge, next to the accessory type if provided
     ///   - accessoryType: The type of accessory that appears on the trailing edge: a disclosure indicator or a details button with an ellipsis icon
-    @objc open func setup(title: String,
+    @objc open func setup(title: String = "",
+                          attributedTitle: NSAttributedString? = nil,
```

Comment:
Seems a bit odd to me to have both the title and attributed title. Would it be worth passing in the string and attributed separately? So that way you're using title either way, and then only sometimes adding a bunch of attributes.

Reply:
I think with the comment in the first thread, I want to leave the original setup method alone. Either I can create another setup method that basically implements this, or maybe I can add support for adding NSAttributedString as a property (kinda like you mentioned)